### PR TITLE
Shim for running gcloud storage: Add rm support

### PR DESCRIPTION
**Pull request by dilipped**

### CODE REVIEW

**Comment by dilipped** [comment]
Kokoro is failing because of an issue with cp and ls tests. I have fixed those in #1455

File: gslib/commands/rm.py [hunk 1]
```diff
@@ -221,6 +223,19 @@ class RmCommand(Command):
       subcommand_help_text={},
   )
 
+  # TODO(b/b/188092601) Add -f after continue-on-error is supported in
```
Comment (rrauber) [inline gslib/commands/rm.py:226]: b/ is repeated here

File: gslib/tests/test_rm.py [hunk 2]
```diff
@@ -61,6 +67,22 @@ def _CleanRmUiOutputBeforeChecking(self, stderr):
...
+    """Remove irrelevant lines from the output lines."""
+    stderr_lines = stderr.splitlines()
+    valid_lines = []
+    strings_to_remove = set(['Removing objects:', 'Removing Buckets:', '  '])
+    for line in stderr_lines:
+      if line in strings_to_remove:
+        continue
+      # Gcloud storage logs a '.' for each resource in non-interactive mode.
+      # This might get added to the start of a valid line.
+      # e.g. .Removing gs://bucket/obj
```
Comment (rrauber) [inline gslib/tests/test_rm.py:80]: Is the period before "Removing" intentional?

**Review by rrauber — APPROVED** [review]

File: gslib/tests/test_rm.py (hunk 2 above)
Comment (dilipped) [reply gslib/tests/test_rm.py]: Yes, it is intentional. I added quotes around the string to make it obvious.

File: gslib/commands/rm.py (hunk 1 above)
Comment (dilipped) [reply gslib/commands/rm.py]: Actually, this entire TODO should be removed because it was already taken care of.

**Pull request merged**